### PR TITLE
Ovidiu Taralesca

### PR DESCRIPTION
**Pull request by taralesca**

### CODE REVIEW

File: orders-service-client/src/main/java/http/GetOrderListClient.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Arrays;
+import java.util.List;
+
+public class GetOrderListClient {
```
Comment (cribratu) [inline orders-service-client/src/main/java/http/GetOrderListClient.java]: You could have used the same client we created together (renamed maybe). It works with the same entity: Order

File: orders-service/src/main/java/com/procourier/handlers/GetOrdersHandler.java [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+
+    public GetOrdersHandler(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+
+    @Override
+    public Object handle(Request request, Response response) throws Exception {
+        response.header("Content-Type", "application/json;charset=utf-8");
+        List<Order> orders = orderService.getOrders();
```
Comment (cribratu) [inline orders-service/src/main/java/com/procourier/handlers/GetOrdersHandler.java]: You could make orders to be final

File: orders-service-client/src/main/java/main/Main.java [hunk 3]
```diff
@@ -2,12 +2,22 @@
...
 import com.procourier.model.Order;
 import http.GetOrderByIdClient;
+import http.GetOrderListClient;
+
+import java.util.List;
 
 public class Main {
     public static void main(String[] args) {
-        GetOrderByIdClient client = new GetOrderByIdClient();
+        GetOrderListClient client = new GetOrderListClient();
```
Comment (cribratu) [inline orders-service-client/src/main/java/main/Main.java]: it's better to make final all the variables

File: orders-service-client/src/main/java/http/GetOrderListClient.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        return null;
+    }
+
```
Comment (cribratu) [inline orders-service-client/src/main/java/http/GetOrderListClient.java]: too many empty lines

**Comment by cribratu** [comment]
Good job! 👍 No need to implement the changes, it was just an example of how things work on a real project.

**Comment by taralesca** [comment]
Thank you for your feedback! I will keep your advice in mind for next time.

**Pull request closed without merging**